guide/12-enrich-data-with-thematic-information/part7_discover_and_enrich_standard_geographies.ipynb

```diff
@@ -0,0 +1,1165 @@
+{
```

Comment:
<p>It is not uncommon to already have standard geography unique identifiers. However, if you need to retrieve those within a larger area, you can use <code>standard_geography_query</code>. The most versatile parameter in this method is <code>geoquery</code>. You can find more information on the available options for the <code>geoquery</code> parameter in the <a href="https://developers.arcgis.com/rest/geoenrichment/api-reference/standard-geography-query.htm#geographyQuery" target="_blank">geographyQuery parameter documentation</a>. We can start by seeing what is returned when searching for <code>seattle</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1405/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>